options/linux/include/sys/epoll.h

```diff
@@ -40,7 +43,11 @@ typedef union epoll_data {
 struct epoll_event {
 	uint32_t events;
 	epoll_data_t data;
-};
+}
```

Comment:
Why not just always packed?

Reply:
My host (glibc, via a test program), musl and bionic seem to agree that this is correct and only necessary for x86_64.